src/code/actions.js

```diff
@@ -95,6 +95,20 @@ function navigateToStartPage(url) {
   };
 }
 
+export function continueFromVictory() {
+  return (dispatch, getState) => {
+    const { routeSpec, authoring, gems } = getState(),
+          currentMission = routeSpec.level + "" + routeSpec.mission,
```

Comment:
I can't think of a specific case where this would fail, but I'd think it should be something like `routeSpec.level + "." + routeSpec.mission` to avoid a weird case where "1.11" = "11.1"